blog/2021/08/DEQ.md

```diff
@@ -0,0 +1,426 @@
+@def rss_pubdate = Date(2021, 1, 18)
+@def rss = """ Implementing Deep Equilibrium Models in Julia... """
+@def published = "18 January 2021"
+@def title = "Implementing Deep Equilibrium Models in Julia"
+@def authors = """Chris Rackauckas, Mike Innes, Yingbo Ma, Jesse Bettencourt, Lyndon White, Vaibhav Dixit"""  
+
+<!-- Translations: [Traditional Chinese](/blog/2019/04/fluxdiffeq-zh_tw) -->
+
+In this blog post we will show you how to easily, efficiently, and
+robustly use steady state solvers with neural networks in Julia.
+
+<!-- ![Flux ODE Training Animation](https://user-images.githubusercontent.com/1814174/51399500-1f4dd080-1b14-11e9-8c9d-144f93b6eac2.gif) -->
+
+The combination of differential equations and neural networks has aroused wide-spread interest
+since 2017. For more on the history of this field, please see the seminal work
+[Neural Ordinary Differential Equations](https://arxiv.org/abs/1806.07366) and our previous
+blog post [DiffEqFlux.jl – A Julia Library for Neural Differential Equations](https://julialang.org/blog/2019/01/fluxdiffeq/).
+In my opinion, this is one of the best, if not
+the most successful, method that interweaves machine learning with another field of research.
+In doing so, not only do we get to reap the benefits of centuries of research in the
+differential equations community, we also shift our perspective from the discrete to the
+continuous for machine learning, and from the explicit to the black-box for differential
```

Comment:
the differential equations are white boxed!

Reply:
I see what you're going for from the paragraph later. So I think should be rephrased though I am not sure how right now.